common/src/main/java/org/ergoplatform/appkit/JavaHelpers.scala

```diff
@@ -77,6 +78,22 @@ object Iso extends LowPriorityIsos {
     override def from(t: (TokenId, Long)): ErgoToken = new ErgoToken(t._1, t._2)
   }
 
+  implicit val isoJListErgoTokenToMapPair: Iso[JList[ErgoToken], Map[ModifierId, Long]] = 
+    new Iso[JList[ErgoToken], Map[ModifierId, Long]] {
+    override def to(a: JList[ErgoToken]): Map[ModifierId, Long] = {
+      import JavaHelpers._
+      a.convertTo[IndexedSeq[(TokenId, Long)]]
+        .map(t => bytesToId(t._1) -> t._2)
+        .toMap
+    }
+    override def from(t: Map[ModifierId, Long]): JList[ErgoToken] = {
+      import JavaHelpers._
+      val pairs: IndexedSeq[(TokenId, Long)] = t.toIndexedSeq
+        .map(t => (Digest32 @@ idToBytes(t._1)) -> t._2)
+      pairs.convertTo[JList[ErgoToken]]
+    }
+  }
```

Comment:
Please use LinkedListMap

Reply:
Oh, I missed this one. Thanks! Fixed.

Reply:
BTW, I assumed you meant `LinkedHashMap`, cause I did not find `LinkedListMap`.

Reply:
yes, sure